lib/ex_doc/language.ex

```diff
@@ -131,6 +131,15 @@ defmodule ExDoc.Language do
               opts: keyword()
             }
 
-  def get(:elixir), do: ExDoc.Language.Elixir
-  def get(:erlang), do: ExDoc.Language.Erlang
+  def get(:elixir, _module), do: {:ok, ExDoc.Language.Elixir}
+  def get(:erlang, _module), do: {:ok, ExDoc.Language.Erlang}
+
+  def get(language, module) do
```

Comment:
If we are not going to inspect the arguments in the warning message, I think it is wise to guard the clause otherwise if users pass `language = "erlang"` he won't be able to tell what is wrong with the arguments from the warning message.

```suggestion
  def get(language, module) when is_atom(language) and is_atom(module) do
```

Reply:
That's reasonable - I hadn't considered it at all.